### DISCUSSION TITLE
Sonoff MiniR4

### DISCUSSION THREAD

**Discussion by XMP40** [RavenSystem/esp-homekit-devices#2138] [MEPLHAA Scripts]
Hello,
has anybody create a HAA confirmation for new SonoffMiniR4?
Would be great in you could share it. 
Thanks

**Comment by tsgxwolf** [comment]
{"c":{"io":[[[0,27],6,1],[[19,26],2]],"l":19,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":26}]},"1":{"r":[{"g":26,"v":1}]},"b":[{"g":0},{"g":27},{"g":27,"t":0}]}]}

from：https://github.com/RavenSystem/esp-homekit-devices/discussions/2069

**Reply by tsgxwolf** [reply]
I would to ask,is there any way to flash miniR4 with ota,just like DIY mode？
